### PR TITLE
Add Fake Cambodian Name

### PR DESCRIPTION
**Pull request by Phanithism**

### CODE REVIEW

File: test/test_name_kh.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+require 'helper'
+
+class TestFakerNameKH < Test::Unit::TestCase
+  def setup
+    @nameKH = FFaker::NameKH
```
Comment (Phanithism) [inline test/test_name_kh.rb]: already changed :)

**Comment by marocchino** [comment]
Thanks!

**Comment by Phanithism** [comment]
@marocchino by the ways, when will you plan to release it on RubyGem ?

**Comment by marocchino** [comment]
@Phanithism Actually I don't have [ownership on this project](https://rubygems.org/gems/ffaker).

**Pull request merged**